doc/source/atomgroup.rst

```diff
@@ -0,0 +1,331 @@
++-------------------------------+------------+----------------------------+
+| ``s.intersection(t)``         | ``s & t``  | new Group with elements    |
+|                               |            | common to ``s`` and ``t``  |
++-------------------------------+------------+----------------------------+
+| ``s.difference(t)``           | ``s - t``  | new Group with elements of |
+|                               |            | ``s`` that are not in ``t``|
++-------------------------------+------------+----------------------------+
+| ``s.symmetric_difference(t)`` | ``s ^ t``  | new Group with elements    |
+|                               |            | that are part of ``s`` or  |
+|                               |            | ``t`` but not both         |
++-------------------------------+------------+----------------------------+
+
+Groupby and split
+-----------------
+
+An :class:`~MDAnalysis.core.groups.AtomGroup` can be constructed from another by separating atoms by properties. 
+
+:meth:`AtomGroup.split <MDAnalysis.core.groups.AtomGroup.split>` can create a list of :class:`~MDAnalysis.core.groups.AtomGroup`\ s by splitting another :class:`~MDAnalysis.core.groups.AtomGroup` by the 'level' of connectivity: one of *atom*, *residue*, *molecule*, or *segment*. 
+
+.. code-block:: python
+
+    >>> ag1 = u.atoms[:100]
+    >>> ag1
+    <AtomGroup with 100 atoms>
+    >>> ag1.split('residue')
+    [<AtomGroup with 19 atoms>,
+    <AtomGroup with 24 atoms>,
+    <AtomGroup with 19 atoms>,
+    <AtomGroup with 19 atoms>,
+    <AtomGroup with 19 atoms>]
+
+
+An :class:`~MDAnalysis.core.groups.AtomGroup` can also be separated according to values of `topology attributes <../topology_system.html#topology-attributes>`_ to produce a dictionary of :code:`{value:AtomGroup}`. 
+
+.. code-block::
+
+    >>> u = mda.Universe(PSF, DCD)
+    >>> u.atoms.groupby('masses')
+    {32.06: <AtomGroup with 7 atoms>, 1.008: <AtomGroup with 1685 atoms>, 12.011: <AtomGroup with 1040 atoms>, 14.007: <AtomGroup with 289 atoms>, 15.999: <AtomGroup with 320 atoms>}
+
+Passing in multiple attributes groups them in order:
+
+.. code-block::
+
+    >>> u.atoms.groupby(['masses', 'resnames'])
+    {(32.06, 'MET'): <AtomGroup with 6 atoms>, (32.06, 'CYS'): <AtomGroup with 1 atom>, (1.008, 'LEU'): <AtomGroup with 176 atoms>, (1.008, 'SER'): <AtomGroup with 25 atoms>, (1.008, 'TYR'): <AtomGroup with 63 atoms>, (1.008, 'ARG'): <AtomGroup with 169 atoms>, (1.008, 'GLU'): <AtomGroup with 108 atoms>, (1.008, 'GLY'): <AtomGroup with 60 atoms>, (1.008, 'ASN'): <AtomGroup with 24 atoms>, ..., }
+
+Constructing from Atoms
+-----------------------
+
+An :class:`~MDAnalysis.core.groups.AtomGroup` can be created from an iterable of :class:`~MDAnalysis.core.groups.Atom`\ s:
+
+.. code-block:: python
+
+    >>> atom1 = u.atoms[4]
+    >>> atom2 = u.atoms[6]
+    >>> atom3 = u.atoms[2]
+    >>> ag = mda.AtomGroup([atom1, atom2, atom3])
```

Comment:
Maybe a cute hack to mention is you can use the built in sum function

Reply:
I didn't know you could do that -- that is cute!